manifests/a/AVerMedia/RECentral/4.7.44.1/AVerMedia.RECentral.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: AVerMedia.RECentral
+PackageVersion: 4.7.44.1
+PackageLocale: en-US
+Publisher: AVerMedia TECHNOLOGIES, Inc.

```

```suggestion
Publisher: AVerMedia TECHNOLOGIES, Inc
```
The Installer has the `.`, but the ARP entry for the program itself does not